packages/.eslintrc.json

```diff
@@ -77,7 +77,7 @@
         "@dimensiondev/string/no-unneeded-to-string": "error",
         "@dimensiondev/type/no-instanceof-wrapper": "error",
         "@dimensiondev/type/no-wrapper-type-reference": "error",
-        "@dimensiondev/unicode/specific-set": "error",
+        "@dimensiondev/unicode/specific-set": "off",
```

Comment:
why?

Reply:
this is security rule

Reply:
Are emojis very common? If not, you can use eslint-disable

Reply:
ping

Reply:
Not only emojis, but also some other symbols like Ξ ₿.

Reply:
cc @septs can we have an allow list for this rule? It will be convenient.

Reply:
ping @septs

Reply:
This rule supports allow list. https://dimensiondev.github.io/eslint-plugin/src/rules/unicode/specific-set

Please allow characters you need to use here.

Note: For the code review perspective, `₿` should not be allowed because it looks visually like `B`